src/builtins/index.ts

```diff
@@ -5,29 +5,33 @@ import {
   JupyterFrontEnd,
   JupyterFrontEndPlugin
 } from '@jupyterlab/application';
+import { ITLabPythonManager } from '../python/manager';
 import { ITLabStoreManager } from '../store/manager';
 import { ITLabCtrlManager } from '../widget-controller/manager';
 import * as controllers from './controllers';
 import * as models from './models';
 
 /**
  * Builtins models plugin.
+ * TODO: make registrations cleaner.
  */
 export const labBuiltinsPlugin: JupyterFrontEndPlugin<void> = {
   id: 'tlab:builtins',
   autoStart: true,
-  requires: [ITLabStoreManager, ITLabCtrlManager],
+  requires: [ITLabStoreManager, ITLabCtrlManager, ITLabPythonManager],
   activate: (
     app: JupyterFrontEnd,
     storeManager: ITLabStoreManager,
-    ctrlManager: ITLabCtrlManager
+    ctrlManager: ITLabCtrlManager,
+    pyManager: ITLabPythonManager
   ) => {
-    // register builtins models
+    // register models in store manager and controller manager
     for (const model of Object.values(models)) {
       storeManager.registerModel(model);
+      ctrlManager.registerModel(model);
```

Comment:
why this double registration is necessary? the control mgr can't get the relevant info from the store mgr?

Reply:
Changed in https://github.com/twiinIT/tlab/pull/7/commits/64124456343488f42879894e54c23615bddfe146